pki/simple_path_builder_delegate.cc

```diff
@@ -123,6 +145,10 @@ bool SimplePathBuilderDelegate::IsPublicKeyAcceptable(EVP_PKEY *public_key,
     return true;
   }
 
+  if (IS_OQS_PKEY(pkey_id)) {
+    return true;
```

Comment:
This seems fishy. Any `pkey_id` is an OQS pkey?

Reply:
pkey_id can be any type of pkey.

`IsPublicKeyAcceptable` will return false if the pkey is not an ECC, RSA, and quantum-safe key.

Reply:
Sincere apologies. I completely mis-read the code. Time for the weekend. Please go ahead and merge!